include/regs.h

```diff
@@ -99,7 +99,7 @@
 #define R_TEXTBOX_HEIGHT                         YREG(23)
 #define R_TEXTBOX_ICON_XPOS                      YREG(71)
 #define R_TEXTBOX_ICON_YPOS                      YREG(72)
-#define R_TEXTBOX_ICON_SIZE                      YREG(75)
+#define R_TEXTBOX_ICON_DIMENSION                 YREG(75)
```

Comment:
I'd probably prefer `Length` over `Dimension`, but it seems fine as is.

Reply:
possibly, though to me length makes it sound like its only for that one side. while this is trying to convey that its the same number for length and width. but yeah i guess its not that clear either way

Reply:
I'm keeping "dimension" until someone else says they prefer "length"